fawltydeps/main.py

```diff
@@ -190,6 +179,12 @@ def print_human_readable(self, out: TextIO, details: bool = True) -> None:
 def build_parser() -> argparse.ArgumentParser:
     """Create the CLI parser."""
     parser, option_group = setup_cmdline_parser(description=__doc__)
+    option_group.add_argument(
+        "--generate-toml-config",
+        action="store_true",
+        default=False,
+        help="Print a TOML config section with the current settings, and exit",
+    )
```

Comment:
> In fact I'm starting to wonder if this maybe should be an action instead of an option

Indeed, current actions require one to choose one and this is additional to them.

>  I'm starting to see parallels between our CLI actions and how much of the Analysis object we want to calculate/populate

:+1: